Src/C/run-time/eif_built_in.h

```diff
@@ -88,7 +88,16 @@ extern "C" {
 #define eif_builtin_IDENTIFIED_ROUTINES_eif_current_object_id(object)	eif_reference_id(object)
 #define eif_builtin_IDENTIFIED_ROUTINES_eif_is_object_id_of_current(object,id) EIF_TEST(eif_id_object(id) == object)
 
-#define eif_builtin_REFLECTOR_c_set_dynamic_type(obj,enc_ftype)		eif_set_dynamic_type(obj,enc_ftype)
+/* REFLECTOR class */
+#define eif_builtin_REFLECTOR_c_set_dynamic_type(obj,enc_ftype)			eif_set_dynamic_type(obj,enc_ftype)
+#define eif_builtin_REFLECTOR_c_new_type_instance_of(type_id)			eif_type_malloc(eif_decoded_type(type_id), 0)
+#define eif_builtin_REFLECTOR_c_new_tuple_instance_of(type_id)			RTLNT(eif_decoded_type(type_id).id)
+#define eif_builtin_REFLECTOR_c_new_instance_of(type_id)			RTLNSMART(eif_decoded_type(type_id).id)
+#define eif_builtin_REFLECTOR_field_count_of_type(type_id)			ei_count_field_of_type(type_id)
+#define eif_builtin_REFLECTOR_field_count_of_type(type_id)			ei_count_field_of_type(type_id)
```

Comment:
Same line as before ...
I am surprised the C compiler did not report any warning .

Reply:
I've compiled it and I didn't notice it but .. good catch I will update it.

Reply:
Done